README.md

```diff
@@ -108,6 +108,29 @@ class Accounts::OwnershipsController < ActionController::Base
 end
 ```
 
+### Standalone Path ###
+
+When one controller action method supports multiple paths, the additional path can be documented by using `@!path` directive.
+
+```ruby
+  # @!path [GET] /extra/accounts/ownerships/{id}
```

Comment:
Can both of these paths contain the method description?

Reply:
I'm gonna poke around more to see if it's possible 👍

Reply:
just tested and confirmed that it works. so adding it as an example!